src/components/views/messages/MFileBody.js

```diff
@@ -102,6 +102,9 @@ export default class MFileBody extends React.Component {
         tileShape: PropTypes.string,
         /* whether or not to show the default placeholder for the file. Defaults to true. */
         showGenericPlaceholder: PropTypes.bool,
+        /* to set source to local file path during export */
+        mediaSrc: PropTypes.string,
```

Comment:
I kind of feel like doing it as part of the export with a string replace is sane as it'll catch other references to the MXC too e.g inline markdown/html links